source/common/quic/active_quic_listener.cc

```diff
@@ -23,6 +23,10 @@
 namespace Envoy {
 namespace Quic {
 
+namespace {
+constexpr uint8_t kEcnMask = 0x03;
```

Comment:
nit: brief comment which explains what this is and where the magic number comes from. (It could also just be `3` instead of `0x03`, or maybe `0b00000011` if we want to explicitly show that this masks the two low bits)